k8s-custom-pipelines.yml

```diff
@@ -0,0 +1,356 @@
+resources:

```

Comment:
Correct, once merged, I will remove this from the `release` branch. This way, future PRs will not capture this file in the changelog and it will not be merged into upstream